docs/guide.md

```diff
@@ -0,0 +1,764 @@
+# Guide to Running E3SM
+
+## About This Guide
+
+This guide is intended to walk users step-by-step through the simulation process -- from setting up a simulation to post-processing, long-term archiving, and publishing it.
+
+## Useful Aliases
+
+Setting some aliases may be useful in running the model. You can edit your bash settings file to add aliases. Run `source` on that file to start using your new aliases. Examples:
+
+- Chrysalis – `source ~/.bashrc`
+- Compy – `source ~/.bash_profile`
+- Perlmutter – `source ~/.bash_profile.ext`
+
+Note that the specific file name may differ amongst machines. For example, it might be named `~/.bashrc.ext`.
+
+### Batch Jobs
+
+To check on all batch jobs:
+
+`alias sqa='squeue -o "%8u %.7a %.4D %.9P %7i %.2t %.10r %.10M %.10l %.8Q %j" --sort=P,-t,-p'`
+
+To check on your batch jobs:
+
+`alias sq='sqa -u $USER'`
+
+The output of `sq` uses several abbreviations: ST = Status, R = running, PD = pending, CG = completing.
+
+### Directories
+
+You will be working in several directories.
+
+<run_scripts_dir>: `${HOME}/E3SM/scripts`
+
+<code_source_dir>: `${HOME}/E3SM/code`
+
+<simulations_dir> differs amongst machines:
+
+- Anvil/Chrysalis (LCRC) – <simulations_dir>: `/lcrc/group/e3sm/<username>/E3SMv2`
+- Compy (PNNL) – <simulations_dir>: `/compyfs/<username>/E3SMv2`
+- Perlmutter (NERSC) – <simulations_dir>: `/global/cfs/cdirs/e3sm/<username>/E3SMv2`
+
+So, it may be useful to set the following aliases:
+
+```shell
+# Model running (1)
+
+alias run_scripts="cd <run_scripts_dir>"
+alias simulations="cd <simulations_dir>"
+```
+
+1. TODO: this is just a demo! But you can have more aliases if you want!
+
+## Configuring the Model Run – Run Script
+
+Start with an example of a run script for a low-resolution coupled simulation:  
+
+We'll use the [template script](https://github.com/E3SM-Project/E3SM/blob/master/run_e3sm.template.sh) (1) in the E3SM repository, which uses Perlmutter.
+{ .annotate }
+
+1. TODO: this is just a demo! It's always good to start from this tried and trusted script!
+
+Create a new run script or copy an existing one (such as the template above). The path to it should be `<run_scripts_dir>/run.<case_name>.sh`
+
+`# Machine and project`
```

Comment:
Not clear why this subsection suddenly appears here.  These are things people need to edit in the runscript?  Say that.

Reply:
Yes, these are the lines from the runscript that need editing. Makes sense -- I'll add a line explaining these are the lines to edit.